backend/utils.js

```diff
@@ -199,6 +199,20 @@ const hyphenatedStringToCamel = string =>
 const formatCurrency = (number, currency = 'GBP') =>
   number ? number.toLocaleString('en-GB', { style: 'currency', currency }) : ''
 
+const capitalizeUppercaseString = string =>
+  string
+    ? string
+        .split(' ')
+        .map(name => capitalize(name))
```

Comment:
we normally just capitalise the first word rather than all of them?

Reply:
Could probably name this better, meant for names rather than anything else.  Will change in next PR as I've still got some stuff to do.